scripts/del-bpfman-qdiscs.sh

```diff
@@ -1,20 +1,25 @@
 #!/bin/bash
 # This script deletes the clsact and ingress qdiscs (if they exist) on all
-# interfaces.  This has the side-effect of removing any tc_dispatchers (or other
-# programs) that happen to be attached to them.  This is intended to be part of
-# completely cleaning up the bpfman state on a system.  Caution should be used if
+# interfaces. This has the side-effect of removing any tc_dispatchers (or other
+# programs) that happen to be attached to them. This is intended to be part of
+# completely cleaning up the bpfman state on a system. Caution should be used if
 # other applications are currently using these qdiscs for other purposes.
 
-interfaces=()
-for iface in $(ifconfig | cut -d ' ' -f1| tr ':' '\n' | awk NF)
-do
-        interfaces+=("$iface")
-done
+delete_qdiscs() {
+    local iface
+    for iface in $(ifconfig | cut -d ' ' -f1 | tr ':' '\n' | awk NF); do
+        # delete the clsact qdisc if it exists
+        sudo tc qdisc del dev "$iface" clsact 2>/dev/null
+        # delete the ingress qdisc if it exists
+        sudo tc qdisc del dev "$iface" ingress 2>/dev/null
+    done
+}
 
-for i in "${interfaces[@]}";
-do
-	# delete the clsact qdisc if it exists
-	sudo tc qdisc del dev $i clsact 2>/dev/null;
-	# delete the ingress qdisc if it exists
-	sudo tc qdisc del dev $i ingress 2>/dev/null;
-done
+# Delete qdiscs in the default namespace
```

Comment:
Based on thread on slack, may need to wrap this logic in a function so it can be called from `sudo ./scripts/setup.sh uninstall`. That may require breaking it across two files so cleanup can be called directly, or we just assume it is only used with `sudo ./scripts/setup.sh uninstall`.

Reply:
I'll add the qdisc cleanup to `./scripts/setup.sh uninstall`, but I may do it in a separate PR.  I think we have an issue in the has_qdisc() code that is independent from the new netns functionality added in this PR.